src/main/java/org/usfirst/frc4904/robot/humaninterface/operators/DefaultOperator.java

```diff
@@ -14,5 +21,19 @@ public DefaultOperator(String name) {
 	}
 
 	@Override
-	public void bindCommands() {}
+	public void bindCommands() {
+		RobotMap.HumanInput.Operator.joystick.button1.onlyWhileHeld(new ElevatorControl());
+		RobotMap.HumanInput.Operator.joystick.button2.whenPressed(new OverrideEnable(RobotMap.Component.fourBar));
+		RobotMap.HumanInput.Operator.joystick.button2.whenReleased(new OverrideDisable(RobotMap.Component.fourBar));
+		RobotMap.HumanInput.Operator.joystick.button3
+			.whenPressed(new RunIfElse(new SolenoidExtend("FourBarUp", RobotMap.Component.fourBar.lever),
+				new SolenoidRetract("FourBarDown", RobotMap.Component.fourBar.lever),
+				RobotMap.Component.fourBar.lever::isExtended));
+		RobotMap.Input.elevatorSwitchBottom.whenPressed(new SingleOp(() -> {
+			RobotMap.Component.elevatorEncoder.resetViaOffset();
+		}));
+		RobotMap.Input.elevatorSwitchTop.whenPressed(new SingleOp(() -> {
```

Comment:
This should not be in default operator. I'd rather it be in Robot.java

Reply:
(note, that isn't a great place for it but it's better. You can make a new file or something and reference it in Robot.java the same way DefaultOperator.bindCommands is called)